domains/hero.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+      "username": "heropj",
+      "email": "pranjalpandey1022003gmail.com"
+    },
+    "record": {
+      "URL": "https://heroportfolio-two.vercel.app/"
```

please add more stuff into your website as it should be developer related

---

hello, that website is my portfolio website.. pls check [/portfolio](https://heroportfolio-two.vercel.app/portfolio) page.

---

> hello, that website is my portfolio website.. pls check [/portfolio](https://heroportfolio-two.vercel.app/portfolio) page.

you've given URL records to redirect to [this](https://heroportfolio-two.vercel.app/) website and not your portfolio website, please change it

```suggestion
      "URL": "https://heroportfolio-two.vercel.app/portfolio"
```